packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -1811,6 +1821,19 @@ export class Combobox
                 />
               )}
             <slot />
+            {this.noMatches === "add" && (
+              <li
```

Comment:
This `<li>` looks good, but I noticed the other items inside the `<ul>` are `<calcite-combobox-item>`s. I wonder about how the accessibility and semantics of that are? Its a bit weird to have the following:

```html
<ul>
  <calcite-combobox-item></calcite-combobox-item>
</ul>
```

https://github.com/Esri/calcite-design-system/blob/390036cca5d200fba39076023e452640b2d4b75f/packages/calcite-components/src/components/combobox/combobox.tsx#L1806-L1831

I'm not sure if we should be using a `calcite-combobox-item` here and removing the `ul`?

Reply:
Ok, I'll open a research issue to investigate. 
 
I  see that slotted elements have already been wrapped in `ul` before the `selectAll` PR https://github.com/Esri/calcite-design-system/pull/11721/files#diff-9bc2dae4daf67ad95f668837f3083752245fbc22e0a58b1e5a043830e05af02cR1811-R1812
 
We're slotting in `combobox-item`s. 
 
Per our discussion, it may be ok to have a mix of `combobox-item`s and `<li>` or `combobox-item`s alone, since they use `<li>` internally, but we should look into it.

Reply:
[**Accessibility and semantics of slotted `calcite-combobox-item`s inside `<ul>`**](https://github.com/Esri/calcite-design-system/issues/12590#top)#12590